auraed/src/runtime/cgroup_table.rs

```diff
@@ -24,17 +24,16 @@ impl CgroupTable {
     /// Returns an error if a duplicate [cell_name] already exists in the cache.
     pub(crate) fn insert(
         &self,
-        cell_name: &str,
-        cgroup: &Cgroup,
+        cell_name: String,
```

Comment:
why this type change? we shouldn't be taking ownership of the cell name so I think this should still be a borrow (from which we can clone) no?

Reply:
I consider taking a reference and then immediately cloning to be "lying" to the caller. Insert needs ownership of `cell_name`, so the signature should represent that. It may also be a scenario where the caller does not need ownership and we can therefore avoid a clone.